### PR TITLE
Include GameDev VM Application files

### PR DESCRIPTION
**Pull request by dciborow**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Adding required files to package Azure GameDev VM as a Solution Template or Managed Application

### CODE REVIEW

**Comment by dciborow** [comment]
@bmoore-msft ,

I am proposing the addition of the files needed to package this app as a Solution Template or Managed App. (Now that we have our ARM template here, we are thinking its best to centralize the last few pieces)

 Does the placement of the additional files make sense? Looking for a way to organize these within the given structure, which I can continue to repeat with future templates.

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: application-workloads/azure-gamedev/gamedev-vm/README.md [hunk 1]
```diff
@@ -10,6 +10,6 @@
 
 ![Bicep Version](https://azurequickstartsservice.blob.core.windows.net/badges/application-workloads/azure-gamedev/gamedev-vm/BicepVersion.svg)
 
-[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fapplication-workloads%2Fazure-gamedev%2Fgamedev-vm%2Fazuredeploy.json)
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fapplication-workloads%2Fazure-gamedev%2Fgamedev-vm%2Fazuredeploy.json2F/createUIDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fapplication-workloads%2Fazure-gamedev%2Fgamedev-vm%2FcreateUiDefinition.json)
```
Comment (dciborow) [inline application-workloads/azure-gamedev/gamedev-vm/README.md:13]:
```suggestion
[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fapplication-workloads%2Fazure-gamedev%2Fgamedev-vm%2Fazuredeploy.json/createUIDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fapplication-workloads%2Fazure-gamedev%2Fgamedev-vm%2FcreateUiDefinition.json)
```

File: application-workloads/azure-gamedev/gamedev-vm/azuredeploy.parameters.json [hunk 2]
```diff
@@ -7,6 +7,123 @@
...
         "passwordAdministratorLogin": {
             "value": "GEN-PASSWORD"
+        },
+        "enableManagedIdentity": {
+            "value": "GEN-UNIQUE"
+        },
+        "virtualMachineName": {
+            "value": "GEN-UNIQUE"
+        },
+        "enableManagedIdentity ": {
```
Comment (dciborow) [inline application-workloads/azure-gamedev/gamedev-vm/azuredeploy.parameters.json:17]:
```suggestion
        "enableManagedIdentity": {
```

File: application-workloads/azure-gamedev/gamedev-vm/azuredeploy.parameters.json [hunk 3]
```diff
@@ -7,6 +7,123 @@
         },
         "passwordAdministratorLogin": {
             "value": "GEN-PASSWORD"
+        },
+        "enableManagedIdentity": {
+            "value": "GEN-UNIQUE"
+        },
```
Comment (dciborow) [inline application-workloads/azure-gamedev/gamedev-vm/azuredeploy.parameters.json:13]:
```suggestion
```

File: application-workloads/azure-gamedev/gamedev-vm/azuredeploy.parameters.json [hunk 4]
```diff
@@ -7,6 +7,120 @@
...
+            "value": ""
+        },
+        "parsec_host": {
+            "value": ""
+        },
+        "parsec_userEmail": {
+            "value": ""
+        },
+        "parsec_isGuestAccess": {
+            "value": ""
```
Comment (dciborow) [inline application-workloads/azure-gamedev/gamedev-vm/azuredeploy.parameters.json:84]:
```suggestion
            "value": false
```

File: application-workloads/azure-gamedev/gamedev-vm/nestedtemplates/gamedev-vm.bicep [hunk 5]
```diff
@@ -1,4 +1,4 @@
-@description('Resource Location.')
+'@description('Resource Location.')
```
Comment (dciborow) [inline application-workloads/azure-gamedev/gamedev-vm/nestedtemplates/gamedev-vm.bicep:1]:
```suggestion
@description('Resource Location.')
```

**Comment by dciborow** [comment]
10:57:24 PM - Error: Code=InvalidTemplate; Message=Deployment template validation failed: 'Template parameter 'numDataDisks' was provided an invalid value. Expected a value of type 'Integer', but received a value of type 'String'. Please see https://aka.ms/resource-manager-parameter-files for usage details.'.

**Pull request closed without merging**